test/igniter/code/within_test.exs

```diff
@@ -0,0 +1,82 @@
+defmodule Igniter.Code.WithinTest do
+  alias Igniter.Code.Common
+  alias Sourceror.Zipper
+  use ExUnit.Case
+  import Igniter.Test
+
+  describe "with within" do
+    test "performs multiple changes to some module" do
+      test_project()
+      |> Igniter.create_new_file("lib/some_module.ex", """
+      defmodule SomeModule do
+        alias Some.Example
+
+        def some_function(a, b) do
+          Example.function(a, b)
+        end
+      end
+      """)
+      |> apply_igniter!()
+      |> Igniter.Project.Module.find_and_update_module!(SomeModule, fn zipper ->
+        zipper
+        |> within!(fn zipper ->
+          pred = &match?(%Zipper{node: {:alias, _, _}}, &1)
```

Comment:
We never really want to match on the zipper node if we can avoid it. For this example I'd pick something exactly matching for simplicity, like `Igniter.Code.Common.nodes_equal?(zipper, Some.Module)`

Reply:
Ah, ignore that, use `Igniter.Code.Function.function_call?(zipper, :alias, 1)

Reply:
Your example using `remove` will actually remove *all* aliases from the module, so that's why you want to be as specific as possible :).

The way I would typically recommend doing it is instead of using remove and add, you would *move* to the location of the node you want to modify and then *replace* that node.

```elixir
case move_to_my_alias(zipper) do
  {:ok, zipper} -> 
    {:ok, Igniter.Code.Common.replace_code(zipper, "import Some.Example, only: [:function]")}
  :error ->
    {:ok, zipper}
end
```

Reply:
I knew my way wasn't the right way 😂 But this way I'll learn the right way. 💯